kdumpctl

```diff
@@ -1338,7 +1335,7 @@ do_estimate()
 		echo -e "Encrypted kdump target requires extra memory, assuming using the keyslot with maximum memory requirement\n"
 	fi
 
-	estimated_size=$((kernel_size + mod_size + initrd_size + runtime_size + crypt_size))
+	estimated_size=$((kernel_size + mod_size + initrd_size * 3 + crypt_size))
```

Comment:
Hi @licliu, can you help me understand me why you triple the initrd_size? Are you assuming the unpacked initramfs will grow twice as big as before at maximum?